### ISSUE TITLE
Unfixed waiting time

### ISSUE THREAD

**Issue by swapnil1010** [opensource-emr/telemedicine#9]
when patient waiting on waiting room he don't have any idea about how much he should wait so we should assign estimated waiting time to perticular patient so it will be satisfiable to patient

**Comment by FaveDoc** [comment]
Yes, depending on how many ppl in the queue, and maybe 10 mins/pt for now.

also planning to add scheduling so the patient can schedule  an appt beforehand
